### PR TITLE
bootstrap project

### PR DESCRIPTION
**Pull request by nd0ut**
What's new:
 - Karma/Jasmine/jest-matchers (instead of jasmine's expect) for in-browser tests, with BrowserStack plugin
 - Jest for node tests
 - So the same tests can be run in both node and browser
 - Rollup serves ESM from Babel and builds ESM and UMD bundles
 - Flow
 - Eslint, Prettier, EditorConfig
 - Readme, Authors
 - ReleaseIt

### CODE REVIEW

File: browserslist-stats.json [hunk 1]
```diff
@@ -0,0 +1 @@
...0.0023514251988130007,"10.0-10.1":0,"11.5":0,"12.1":0.0032919952783382004},"ios_saf":{"8":0.009640843315133303,"3.2":0,"4.0-4.1":0.00023514251988130007,"4.2-4.3":0.00023514251988130007,"5.0-5.1":0.005878562997032501,"6.0-6.1":0.007054275596439002,"7.0-7.1":0.009640843315133303,"8.1-8.4":0.012227411033827603,"9.0-9.2":0.15119664028367594,"9.3":0.08182959691869242,"10.0-10.2":0.28005474117862833,"10.3":1.3680591806694036,"11.0-11.2":0.5356546602896015},"op_mini":{"all":0.16742147415548564},"android":{"3":0.00023514251988130007,"4":0.005643420477151201,"62":0.15966177099940274,"2.1":0,"2.2":0.00047028503976260014,"2.3":0.003527137798219501,"4.1":0.0009405700795252003,"4.2-4.3":0.0049379929175073015,"4.4":0.023514251988130005,"4.4.3-4.4.4":0},"bb":{"7":0.00047028503976260014,"10":0.0023514251988130007},"op_mob":{"12":0.018576259070622707,"37":0.018576259070622707,"12.1":0.018576259070622707},"and_chr":{"62":2.7224800951856922},"and_ff":{"57":0.07265903864332172},"ie_mob":{"10":0,"11":0},"and_uc":{"11.4":0.40515056175548003},"samsung":{"4":0,"5":0,"6.2":0},"and_qq":{"1.2":0},"baidu":{"7.12":0}},"meta":{"start_date":"2017-06-23","end_date":"2017-12-23"},"uid":"custom.49222282|undefined"}
```
Comment (Zmoki) [inline browserslist-stats.json]: Where did you get this data? From our site?

File: .babelrc [hunk 2]
```diff
@@ -0,0 +1,16 @@
+{
+  "presets": [
+    [
+      "@babel/preset-env",
+      {
+        "modules": false,
+        "targets": {
+          "browsers": ["> 1% in my stats", "not ie <= 11"],
```
Comment (Zmoki) [inline .babelrc]: defaults `> 0.5%, last 2 versions, Firefox ESR, not dead` will be better

File: karma.conf.js [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+        browser: 'Firefox',
+        browser_version: '59.0',
+        os: 'Windows',
+        os_version: '10',
+      },
+    },
+
+    client: {jasmine: {}},
+  })
+}
```
Comment (Zmoki) [inline karma.conf.js]: Can you rename to `karma.config.js`?

File: browserslist-stats.json (hunk 1 above)
Comment (nd0ut) [reply browserslist-stats.json]: @Zmoki from webclient-frontend repo

**Comment by Zmoki** [comment]
@nd0ut добавь потом файл browserstack.example.json создай

**Pull request merged**